app/routes/questions/add.tsx

```diff
@@ -1,86 +1,112 @@
-    } else {
-        relatedQuestions = formData.getAll('relatedQuestion').map(question => ({ title: question }));
-    }
+  const result = await addQuestion(title, relatedQuestions)
+  console.log('Added question "' + title + '", response:', result)
 
-    // Make sure the question is formatted as a question
-    if (!title.endsWith('?')) title = title + '?'
-    title = title[0].toUpperCase() + title.substring(1);
+  return redirect(redirectTo)
+}
 
-    const result = await addQuestion(title, relatedQuestions);
-    console.log('Added question "' + title + '", response:', result);
+type Props = {
+  title: string
+  relatedQuestions: string[]
+}
 
-    return redirect(redirectTo);
-};
+export const AddQuestion = ({title, relatedQuestions}: Props) => {
+  const [remixSearchParams] = useSearchParams()
+  const [stateString] = useState(() => remixSearchParams.get('state') ?? '')
+  const [isSubmitted, setSubmitted] = useState(false)
 
-export const AddQuestion = ({ title, relatedQuestions, onQuestionAdded }) => {
-    const actionData = useActionData<typeof action>();
-    const [remixSearchParams] = useSearchParams();
-    const [stateString] = useState(() => remixSearchParams.get('state'));
+  const submit = useSubmit()
 
-    const submit = useSubmit();
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    submit(e.currentTarget, {replace: true})
+    setSubmitted(true)
+  }
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        submit(e.currentTarget, { replace: true });
-        onQuestionAdded(title);
-    }
+  useEffect(() => {
+    setSubmitted(false)
+  }, [title])
 
+  if (isSubmitted) {
     return (
-        <Form method="post"
-              action="/questions/add"
-              className="result-item none-of-the-above"
-              title="Request a new question"
-              onSubmit={handleSubmit}
-        >
-            <input type="hidden" name="title" value={title}/>
-            <input type="hidden" name="stateString" value={stateString}/>
-            {relatedQuestions.map(title => (
-                <input type="hidden"
-                       name="relatedQuestion"
-                       key={title}
-                       value={title}
-                />
-            ))}
-            <button type="submit" className="transparent-button result-item">
-                ＋ None of these: Request an answer to my exact question above
-            </button>
-        </Form>
+      <p className="result-item none-of-the-above">
+        Thanks for asking a new question! {title} was added to our suggestion box. It might take a
+        while for it to be answered by our writers, but check back in a few months.
+        <br />
+        <br />
+        The list of current suggestions can be found at{' '}
+        <a href="https://coda.io/@alignmentdev/ai-safety-info/suggested-questions-66">
```

Comment:
this link could be moved into the previous text, e.g. "added to our <a>suggestion box</a>".
I only did it this way because alerts don't allow html

Reply:
can be changed on hackathon 👍